docs/deploy-cluster.md

```diff
@@ -0,0 +1,150 @@
+`load_config_interval_secs` | 2 * 60                   | Load config interval.
+`meta_ingest_thread_num`    | 3.                       | Meta daemon's ingest thread number.
+
+**Storage Service** support the following config properties.
+
+Property Name                       | Default Value              | Description
+----------------------------------- | -------------------------- | -----------
+`port`                              | 44500                      | Storage daemon listening port.
+`reuse_port`                        | true                       | Whether to turn on the SO_REUSEPORT option.
+`data_path`                         | ""                         | Root data path, multi paths should be split by comma. For rocksdb engine, one path one instance.
+`local_ip`                          | ""                         | IP address which is used to identify this server, combined with the listen port.
+`mock_server`                       | true                       | Start mock server.
+`daemonize`                         | true                       | Whether to run the process as a daemon.
+`pid_file`                          | "pids/nebula-storaged.pid" | File to hold the process id.
+`meta_server_addrs`                 | ""                         | List of meta server addresses, the format looks like ip1:port1, ip2:port2, ip3:port3.
+`store_type`                        | "nebula"                   | Which type of KVStore to be used by the storage daemon.Options can be \"nebula\", \"hbase\", etc.
+`num_io_threads`                    | 16                         | Number of IO threads
+`storage_http_thread_num`           | 3                          | Number of storage daemon's http thread.
+`num_worker_threads`                | 32                         | Number of workers.
+`engine_type`                       | rocksdb                    | rocksdb, memory...
+`custom_filter_interval_secs`       | 24 * 3600                  | interval to trigger custom compaction.
+`num_workers`                       | 4                          | Number of worker threads.
+`rocksdb_disable_wal`               | false                      | Whether to disable the WAL in rocksdb.
+`rocksdb_db_options`                | ""                         | DBOptions, each option will be given as <option_name>:<option_value> separated by.
+`rocksdb_column_family_options`     | ""                         | ColumnFamilyOptions, each option will be given as <option_name>:<option_value> separated by.
+`rocksdb_block_based_table_options` | ""                         | BlockBasedTableOptions, each option will be given as <option_name>:<option_value> separated by.
+`batch_reserved_bytes`              | 4 * 1024                   | Default reserved bytes for one batch operation
+`block_cache`                       | 4                          | BlockBasedTable:block_cache : MB
```

Comment:
I think this is also a rocksdb config, right?

It would be great to make all rocksdb configuration start with ```rocksdb_```